map.go

```diff
@@ -131,7 +132,7 @@ func (r *mapRenderer) finalize() {
 	sort.Slice(
 		r.pairs,
 		func(i, j int) bool {
-			return r.pairs[i].Key < r.pairs[j].Key
+			return natsort.Less(r.pairs[i].Key, r.pairs[j].Key)
```

Comment:
We just need some tests to ensure that the map rendering itself is sorting how we intend.

Reply:
cool, can do.

Reply:
All right the first iteration of tests are in.